main_pullrequest.go

```diff
@@ -47,6 +47,8 @@ func retryOnError(args retryParams) error {
 	return err
 }
 
+var regexpTitleOptions = regexp.MustCompile(`^\[([^\]]+)\].+$`)
```

Comment:
I am not going to block it by this, but a simple tokenizer , or: 1st word with separator ']', second word from that with separator '[', and then split wrt to ',' would be easy, and support `[noci,fast,pass-please]` schema. up to you.

Reply:
Thought a simple regex would be easier to read, but I can write a simple function for parsing the title instead if preferable.

Reply:
I refactored the bit gathering the options. I'm using space separated options (`strings.Fields`) instead of comma separated if you don't mind.